### PR TITLE
feat: add rax-tab-panel component

### PR DESCRIPTION
**Pull request by huxiaoqi567**
*Before* submitting a pull request, please make sure the following is done...

1. Fork the repo and create your branch from `master`.
2. If you've added code that should be tested, add tests!
3. If you've changed APIs, update the documentation.
4. Ensure the test suite passes (`npm test`).
5. Make sure your code lints (`npm run lint`) - we've done our best to make sure these rules match our internal linting guidelines.

### CODE REVIEW

File: examples/components/TabPanelDemo.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
+import {Component, createElement, render } from 'rax';
+import {TabController, TabPanel} from '../../packages/rax-tab-panel/src/';
+import Text from '../../packages/rax-text/src/';
```
Comment (yuanyan) [inline examples/components/TabPanelDemo.js]: 使用包名来引入，不要使用相对路径

File: packages/rax-tab-panel/src/BaseView.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
+/** @jsx createElement */
+
+'use strict';
+
+import {Component} from 'rax';
+import _ from 'simple-lodash';
```
Comment (yuanyan) [inline packages/rax-tab-panel/src/BaseView.js]: 引入lodash的必要性？

File: packages/rax-tab-panel/src/DefaultView.js [hunk 3]
```diff
@@ -0,0 +1,342 @@
...
+        }
+      }
+      this.onPanCallback(e);
+    });
+
+    this.token = res && res.token;
+  }
+
+  onPanCallback = (e) => {
+    // fix: 修复安卓手淘7.5.0下 pan手势被cancel的bug
```
Comment (yuanyan) [inline packages/rax-tab-panel/src/DefaultView.js]: 注释全部使用英文

File: packages/rax-tab-panel/src/BaseView.js (hunk 2 above)
Comment (huxiaoqi567) [reply packages/rax-tab-panel/src/BaseView.js]: 这个是我自己封的lodash  很小很小

Comment (huxiaoqi567) [reply packages/rax-tab-panel/src/BaseView.js]: 已经去除simple-lodash依赖

File: packages/rax-tab-panel/src/SliderView.js [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+    if (this.refs.container) {
+      this.setState({
+        index
+      });
+    }
+  }
+
+
+  componentDidMount() {
+    // TODO extraBindingProps 需要Slider实现ScrollableProtocol
```
Comment (yuanyan) [inline packages/rax-tab-panel/src/SliderView.js]: 避免使用中文注释

**Pull request merged**